lib/gamedata/monster.txt

```diff
@@ -11499,7 +11475,6 @@ spells:BLIND | WOUND | CONF | HEAL | SCARE
 spells:SLOW | WEAVE
 spells:BR_DARK | BR_POIS
 spells:S_KIN
-friends:80:4d4:uruk
 friends:100:3d7:mirkwood spider
 desc:Shelob is an enormous bloated spider, the last daughter of Ungoliant the
```

Comment:
I can suggest to not call Shelob "last daughter"..Probably, something like "one of daughters"